<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Testing</class>
 <widget class="QWidget" name="Testing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Testing</string>
  </property>
  <widget class="QPushButton" name="test_settingsButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test_startButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начать тест</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="questionText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>951</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>700</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainAnswer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="answerSumbitButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>270</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести ответ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testprevQuestionButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>340</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testprevQuestionButton_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>340</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>=&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test_endButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Завершить тест</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>700</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>700</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>700</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>590</y>
     <width>700</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>650</y>
     <width>700</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>710</y>
     <width>700</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>770</y>
     <width>700</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="test_menuButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Главное меню</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plain1">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>490</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plain2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>590</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plain3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>680</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_9">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>740</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="test_settingsButton_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>780</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить настройки</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainLabel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>450</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Количество вопросов в тесте</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainLabel_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>550</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Сложность теста(1-3)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainLabel_3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>640</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Таймер(минут)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainLabel_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>730</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Контрольный режим</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>830</y>
     <width>700</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plain0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
